images/server/Dockerfile.fedora

```diff
@@ -1,6 +1,6 @@
 FROM quay.io/samba.org/sambacc:latest AS builder
-ARG SAMBACC_VER=a2cfac7ae49e
-ARG SAMBACC_REPO=
+ARG SAMBACC_VER=b279787f9550
+ARG SAMBACC_REPO=https://github.com/samba-in-kubernetes/sambacc
```

Comment:
What's the relevance of this added line in this commit?

Reply:
Technically leaving `SAMBACC_REPO` blank is the same as setting it, but I started getting mixed up when working on stuff so I thought it would be better to spell it out explicitly.

Reply:
👍